.github/workflows/buf-pull-request.yml

```diff
@@ -0,0 +1,15 @@
+name: Buf (pull request)
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: bufbuild/buf-setup-action@v1
+    - uses: bufbuild/buf-lint-action@v1
+      with:
+        input: proto
+    # - uses: bufbuild/buf-breaking-action@v1
+    #   with:
+    #     input: proto
+    #     against: 'https://github.com/${GITHUB_REPOSITORY}.git#branch=main'
```

Comment:
Yes, but I thought we should only do that when we hit `v1` right? The whole point of being in `v1beta1` is that we want to make backward-incompatible changes. It sounds like we can ignore certain services or patterns from `buf.yaml`, so that's something we could be doing. I'm realizing though that commented code is ugly, I will replace it with a TODO note instead!